cgi/auth.pl

```diff
@@ -55,6 +55,11 @@
 		user => {
 			email => $User{email},
```

Comment:
Shan't we remove it then ?

Reply:
@alexgarel It might be used by the mobile app, I'm not really sure... https://github.com/openfoodfacts/openfoodfacts-server/pull/7455

Reply:
I'm removing the email and asking about it in https://github.com/openfoodfacts/openfoodfacts-dart/issues/893#issuecomment-1999390759